Dockerfile

```diff
@@ -18,15 +18,16 @@ COPY apis/ apis/
 COPY controllers/ controllers/
 COPY internal/ internal/
 COPY pkg/ pkg/
+COPY versions.txt versions.txt
```

Comment:
nit: Does this need to be copied in? Is it used as part of the docker build?

Reply:
I took it from upstream, not sure why it needs to be copied either since the version numbers are coming from the arguments. But I left it for consistency